src/routes/spent/controller/ExpenseController.ts

```diff
@@ -0,0 +1,23 @@
+import CreateExpenseService from '../services/CreateExpenseService'
+import {Request, Response} from 'express'
+
+class ExpensesController {
```

Comment:
Acho que seria melhor deixar as dependências dessa classe explícitas, injetando através do construtor:

```suggestion
class ExpensesController {  
  constructor(private createExpenseService: CreateExpenseService){}
```